### PR TITLE
Libraries installation in precourse

### PR DESCRIPTION
**Pull request by nimarafati**
Hi,

I added a short video demonstrating how to install the libraries by knitting an Rmd file which: 
- Install libraries from a vector.  
- Check the installation and print confirmation or report which libraries have not been installed.

### CODE REVIEW

File: home_precourse.Rmd [hunk 1]
```diff
@@ -61,11 +61,26 @@ Go to [r-project.org](https://cloud.r-project.org/) and choose the link appropri
 
 RStudio provides you with tools like code editor with highlighting, project management, version control, package building, debugger, profiler and more. Start up RStudio to check that everything has been installed correctly. If not, contact us and we can help you out.
 
+You can also watch [this video](https://youtu.be/suX6nsSUXDw?si=FbaiaCpRNvtjxt5N) for an introduction to using RStudio
+
 `r fa1("chevron-circle-right")` &nbsp; **Install R packages**
 
-Extra R packages used in the workshop exercises (if any) are listed below. It is recommended that you install this in advance. Simply copy and paste the code into R.
+Please download this [test_installation.Rmd file](../test_installation.Rmd) and Knit/Render it.
```
Comment (RedondoMA) [inline home_precourse.Rmd:68]:
```suggestion
Please download this [test_installation.Rmd file](../test_installation.Rmd) and Knit/Render it by clicking on the knit/render icon on the upper right of R Studio or by navigating to file > knit document.   
```

File: home_precourse.Rmd [hunk 2]
```diff
@@ -61,11 +61,26 @@ Go to [r-project.org](https://cloud.r-project.org/) and choose the link appropri
...
+
 `r fa1("chevron-circle-right")` &nbsp; **Install R packages**
 
-Extra R packages used in the workshop exercises (if any) are listed below. It is recommended that you install this in advance. Simply copy and paste the code into R.
+Please download this [test_installation.Rmd file](../test_installation.Rmd) and Knit/Render it.   
+After rendering if everything goes well you should see this message towards end of the rendered html file:  
+
+- **Congratulations you have installed all the packages**.  
+
+If there are any missing package they will be printed at the end of the rendered html file.
```
Comment (RedondoMA) [inline home_precourse.Rmd:73]:
```suggestion
If there are any missing packages they will be printed at the end of the rendered html file.  
```

**Review by RedondoMA — APPROVED** [review]
Perfect!

Cool that they will test the knitting and install all the packages at the same time!

**Review by gbdias — APPROVED** [review]
Looks good!

**Pull request merged**